axlearn/experiments/text/gpt/fuji.py

```diff
@@ -473,7 +504,9 @@ def model_config(
         ffn_dim = scaled_hidden_dim(scale=8 / 3, round_up_to_multiples_of=256)
     if num_kv_heads:
         atten_cfg = GroupedQueryAttention.default_config()
-        atten_input_linear = FusedGroupedQKVLinear.default_config().set(num_kv_heads=num_kv_heads)
+        backend = jax.default_backend()
```

Comment:
The fuji config should not depend on `jax.default_backend()`, otherwise the golden configs will not reflect the actual config being used.

Instead, we can create separate configs for a backend that requires different settings.

Reply:
+1, please follow this example instead if you really need to overwrite some configs, you can add another custom `LayerConfigModifier`like this one: https://github.com/apple/axlearn/blob/main/axlearn/common/trainer_config_modifier.py#L69,

Reply:
Thanks for the review, will update the PR with a custom LayerConfigModifier.